sdk/src/client/node_api/indexer/routes.rs

```diff
@@ -23,6 +24,23 @@ use crate::{
 // hornet: https://github.com/gohornet/hornet/blob/develop/plugins/indexer/routes.go
 
 impl ClientInner {
+    // RouteOutputs is the route for getting basic, alias, nft and foundry outputs filtered by the given
```

Comment:
I feel like something went wrong with this comment

Reply:
I'm still confused, like what is `RouteOutputs`

Reply:
/outputs

Reply:
....

Reply:
I just copied from here https://github.com/iotaledger/inx-indexer/blob/03320ba382e7441e70ddea352011eea9d2e006fc/pkg/server/routes.go#L18, don't know what's your issue with it

Reply:
But they actually have a `RouteOutputs` variant 😆

Reply:
Changed it, okay now?